src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint.py

```diff
@@ -585,13 +585,14 @@ def prepare_latents(
                 "Since strength < 1. initial latents are to be initialised as a combination of Image + Noise."
                 "However, either the image or the noise timestep has not been provided."
             )
+        
+        noise = randn_tensor(shape, generator=generator, device=device, dtype=dtype)
```

Comment:
I guess this is not necessary; I had thought it might cover some cases when `latents is not None` and `return_noise == True` for `self.prepare_latents`, but I guess the `noise` is already initialized properly:

https://github.com/huggingface/diffusers/blob/5175d3d7a542ed1175764a4945f4d8f1ab4fc4be/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint.py#L599-L601